File: EASSET.Web/Modules/ServerTypes/MasterData/OPDColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { OPDRow } from "./OPDRow";

export interface OPDColumns {
    UnitNama: Column<OPDRow>;
    UnitKode: Column<OPDRow>;
    Alamat: Column<OPDRow>;
}

export class OPDColumns extends ColumnsBase<OPDRow> {
    static readonly columnsKey = 'MasterData.OPD';
    static readonly Fields = fieldsProxy<OPDColumns>();
}

File: EASSET.Web/Modules/MasterData/OPD/OPDColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace EASSET.MasterData.Columns;

[ColumnsScript("MasterData.OPD")]
[BasedOnRow(typeof(OPDRow), CheckNames = true)]
public class OPDColumns
{
    //[EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    //public string UnitId { get; set; }
    [EditLink]
    public string UnitNama { get; set; }
    public string UnitKode { get; set; }
    //public string ThAng { get; set; }
    public string Alamat { get; set; }
    //public string Op { get; set; }
    //public DateTime Lu { get; set; }
    //public string Pc { get; set; }
    //public bool Dlt { get; set; }
    //public string Npwp { get; set; }
    //public short IsBlud { get; set; }
    //public int NoUrut { get; set; }
}